lib/streamlit/hello.py

```diff
@@ -14,57 +14,243 @@
 # limitations under the License.
 
 """A "Hello World" app."""
+from __future__ import division
 
 import streamlit as st
-import numpy as np
-import time
+import inspect
+from collections import OrderedDict
+import urllib
 
-
-def run():
-    st.title("Hello Streamlit!")
-    st.write(
+def intro():
+    st.markdown(
         """
-        This is an example **Streamlit** app.
+## Intro...
 
-        If this is your first time using Streamlit, welcome! You may want to
-        take a look at our
-        [tutorials and documentation](https://streamlit.io/docs) next.
-        And feel free to ask us questions any time at
-        [help@streamlit.io](mailto:help@streamlit.io).
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+text text text text text text text text text text text text text text text text
+"""
+    )
 
-        To celebrate this magic moment, below we're generating a bunch of
-        random numbers in a loop for around 10 seconds. Enjoy!
+
+def mapping_demo():
```

Comment:
Make this a private function (i.e. prefix with underscore).

Same with any top-level function here that doesn't get used outside this file.